heartbeat/podman

```diff
@@ -376,6 +376,11 @@ run_new_container()
 		# with the same name, even though podman itself thinks there
 		# is no such container. If so, purge the storage layer to try
 		# to clean the corruption and try again.
+                if [ -n $(echo "$out" | grep -q "unknown*flag") ]; then
```

Comment:
you could do as well w/o a subshell:
```
if echo "$out" | grep -q "unknown*flag"; then

Reply:
i try this before   , but its not work  on my pc ,its there some problem in my system?

Reply:
Seems like you need to use `.*` instead of `*`.

Reply:
my fault  ，its work，thanks ,i commit again can you hellp to review it

Reply:
its done